ocaml/idl/datamodel_errors.ml

```diff
@@ -620,7 +620,7 @@ let _ =
   error Api_errors.vm_requires_iommu ["host"]
     ~doc:"You attempted to run a VM on a host which doesn't have I/O virtualization (IOMMU/VT-d) enabled, which is needed by the VM." ();
   error Api_errors.vm_host_incompatible_version_migrate ["host"; "vm"]
-    ~doc:"You attempted to migrate a VM to a destination host which is older than the source host." ();
+    ~doc:"Cannot migrate a VM to a destination host which is older than the source host." ();
```

Comment:
This uses "host" where the preferred term seems to be "server". Or is "host" specifically used in the context of migration?

Reply:
Hmm, no it's simpler than this, this belongs to the ones that have escaped our attention. We have converted host to server only in the overrides and this was not overriden (which is why I want to reduce the overrides to the bare minimum: if something needs rephrasing, there should be one place to do it, the API source).

Reply:
Shall we merge and fix this some other time or would you like to change it?

Reply:
Let's merge it now; there are more tweaks we can make in a future PR, but right now the objective of this PR is to strip down the SDK overrides.